src/test/java/Homework21.java

```diff
@@ -0,0 +1,52 @@
+public class Homework21 extends LoginTests{
+
+
+    import org.openqa.selenium.By;
+import org.openqa.selenium.Keys;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedCondition;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.testng.Assert;
+import org.testng.TestNG;
+import org.testng.annotations.Test;
+
+    public class Homework20 extends LoginTests {
+
+    }
+//Prerequisite - at least one user created playlist
+
+        String newPlaylistName = "Sample Edited Playlist";
+
+        @Test
+        public void renamePlaylist() {
+
+            String updatedPlaylistMsg = "Updated playlist \"Sample Edited Playlist.\"";
+
+            provideEmail ("demo@class.com");
+            providePassword ("te$t$tudent");
+            clickSubmit ();
```

Comment:
I see you are using the functions provideEmail, providePassword, etc in LoginPage. So you need to import Login Page then you can call these funcs.
//HINT:
![image](https://github.com/Test-Pro-Bootcamp/A33-QA-Automation-Framework/assets/144403954/c55b43d4-d731-4372-acdb-e44c399d7f43)